### PR TITLE
Adds forgotten kit to armament beacon

### PR DESCRIPTION
**Pull request by Coastfront**
## About The Pull Request
Adds the forgotten kit to the armament beacon, holding the same stats as knight armor and stone sentinel, But with the sprites for the old clockwork armor and helmet, as they are very nice looking sprites. Moves said sprites to the suit and hat file respectively. as-well as adding forgotten helmet/armour to the game, which are said armors.
forgottenkit
pictured above

## Why It's Good For The Game
Makes it easier to do any clockcult themed gimmick as the chaplain without wearing the god awful craftable armor, uses sprites that really shoulden't die on the same hill as clockcult. a few others have also told me they'd love to see the old clockcult armor and helmet sprites used for something, as they look nice.

## Changelog
:cl:
add: forgotten kit
add: forgotten armor and helmet.
/:cl:

### CODE REVIEW

**Comment by Winterous** [comment]
This is the same sprite as the old mining reward Inquisitor suit, right? Loved that one.

**Comment by Coastfront** [comment]
This is the same sprites as the inquisitor suit yes, but bronze. Since inquisitor suit sprite iirc was recolored clock cult sprites.

**Comment by ShibaInuLord** [comment]
the helmet looks a little too big on the ground

**Comment by D4C-420** [comment]
Was not expecting the inquisition here. Always enjoyed the Warhammer themed sprites.

**Review by Twaticus — APPROVED** [review]

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
 
+/obj/item/clothing/head/helmet/chaplain/clock
+	name = "forgotten helmet"
+	desc = "It has the unyielding gaze of a god eternally forgotten."
+	icon_state = "clockwork_helmet"
+	inhand_icon_state = "clockwork_helmet_inhand"
+	armor = list(MELEE = 50, BULLET = 10, LASER = 10, ENERGY = 10, BOMB = 0, BIO = 0, RAD = 0, FIRE = 80, ACID = 80)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDESNOUT
+	flags_cover = HEADCOVERSEYES | HEADCOVERSMOUTH
+	strip_delay = 80
```
Comment (Rohesie) [inline code/game/objects/items/holy_weapons.dm:11]:
```suggestion
	strip_delay = 8 SECONDS
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-5). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**